### DISCUSSION TITLE
Maintenance Status and Update Availability of Soduto Project

### DISCUSSION THREAD

**Discussion by avichou** [sannidhyaroy/Soduto#12] [Q&A] [answered]
since soduto main project seems to be dead, and that one hasnt seen an update in months, im wondering if the project is still maintained 

also , is there a way to check for updates, or install this app via homebrew ? 

or do we have to go on this link to check for latest releases every couple of weeks ?

**Comment by avichou** [comment]
i see that there is a homebrew cask however it seems to be the outdated stable release, not this nightly build

**Comment by sannidhyaroy** [comment, marked as answer]
The homebrew version of Soduto is outdated and it's not maintained by the official developer of Soduto. Regarding the status of this project, it seems to be dead officially but there are minor updates being pushed on the official repo of Soduto or by other forks. I have resolved all the major issues with Soduto in my fork, including replacing all deprecated codes and resolving issues regarding connection and compatibility with newer systems. Though I can’t guarantee any Feature Drops to Soduto, but since I personally use the app everyday, you can at least except minor stability updates.

With that being said, I'm not a Apple or Swift Developer in general and this is my very first project in Swift without knowing anything about Swift in the first place. I had prior experience with C#, Java, JavaScript and Dart and just saw the syntax of the language somewhat relatable. Hence, I won't be able to resolve major issues or add features that require expertise in this field.

**Comment by sannidhyaroy** [comment]
> also , is there a way to check for updates, or install this app via homebrew ?
> 
> or do we have to go on this link to check for latest releases every couple of weeks ?

The official version of Soduto doesn't have a update mechanism built-in, so you're supposed to check new updates manually.

The forked version of Soduto in my repository will have an update mechanism starting Soduto v2.0.0 Nightly, so updates will be pushed directly from now on. However, Soduto will not install new updates automatically without the user's explicit consent. When you launch the app for the second time, it'll ask if you want it to check for new releases automatically or not. I'll try to expand on the functionality in future releases.
